CompletedProcess(args=['python', 'temp_test_case.py'], returncode=1, stdout=b'', stderr=b'FFFF..F.FF.F.FFFFFFFF...FFFFFFFFF.FF...F.FFFF..FFFF.F...FFF.FFFFFF.FFFFFFF.....F..FF.FF...FF.FFF.F..\n======================================================================\nFAIL: test0 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 12, in test0\n    self.assertEqual(patched_source(input_0), original_source(input_0))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test1 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 18, in test1\n    self.assertEqual(patched_source(input_1), original_source(input_1))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test10 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 72, in test10\n    self.assertEqual(patched_source(input_10), original_source(input_10))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test11 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 78, in test11\n    self.assertEqual(patched_source(input_11), original_source(input_11))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test14 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 96, in test14\n    self.assertEqual(patched_source(input_14), original_source(input_14))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test16 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 108, in test16\n    self.assertEqual(patched_source(input_16), original_source(input_16))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test17 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 114, in test17\n    self.assertEqual(patched_source(input_17), original_source(input_17))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test19 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 126, in test19\n    self.assertEqual(patched_source(input_19), original_source(input_19))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test20 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 132, in test20\n    self.assertEqual(patched_source(input_20), original_source(input_20))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test21 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 138, in test21\n    self.assertEqual(patched_source(input_21), original_source(input_21))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test22 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 144, in test22\n    self.assertEqual(patched_source(input_22), original_source(input_22))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test23 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 150, in test23\n    self.assertEqual(patched_source(input_23), original_source(input_23))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test24 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 156, in test24\n    self.assertEqual(patched_source(input_24), original_source(input_24))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test25 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 162, in test25\n    self.assertEqual(patched_source(input_25), original_source(input_25))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test26 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 168, in test26\n    self.assertEqual(patched_source(input_26), original_source(input_26))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test27 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 174, in test27\n    self.assertEqual(patched_source(input_27), original_source(input_27))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test30 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 192, in test30\n    self.assertEqual(patched_source(input_30), original_source(input_30))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test31 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 198, in test31\n    self.assertEqual(patched_source(input_31), original_source(input_31))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test32 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 204, in test32\n    self.assertEqual(patched_source(input_32), original_source(input_32))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test33 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 210, in test33\n    self.assertEqual(patched_source(input_33), original_source(input_33))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test34 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 216, in test34\n    self.assertEqual(patched_source(input_34), original_source(input_34))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test35 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 222, in test35\n    self.assertEqual(patched_source(input_35), original_source(input_35))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test36 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 228, in test36\n    self.assertEqual(patched_source(input_36), original_source(input_36))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test37 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 234, in test37\n    self.assertEqual(patched_source(input_37), original_source(input_37))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test38 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 240, in test38\n    self.assertEqual(patched_source(input_38), original_source(input_38))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test4 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 36, in test4\n    self.assertEqual(patched_source(input_4), original_source(input_4))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test40 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 252, in test40\n    self.assertEqual(patched_source(input_40), original_source(input_40))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test44 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 276, in test44\n    self.assertEqual(patched_source(input_44), original_source(input_44))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test46 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 288, in test46\n    self.assertEqual(patched_source(input_46), original_source(input_46))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test47 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 294, in test47\n    self.assertEqual(patched_source(input_47), original_source(input_47))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test48 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 300, in test48\n    self.assertEqual(patched_source(input_48), original_source(input_48))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test49 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 306, in test49\n    self.assertEqual(patched_source(input_49), original_source(input_49))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test51 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 318, in test51\n    self.assertEqual(patched_source(input_51), original_source(input_51))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test52 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 324, in test52\n    self.assertEqual(patched_source(input_52), original_source(input_52))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test53 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 330, in test53\n    self.assertEqual(patched_source(input_53), original_source(input_53))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test54 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 336, in test54\n    self.assertEqual(patched_source(input_54), original_source(input_54))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test56 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 348, in test56\n    self.assertEqual(patched_source(input_56), original_source(input_56))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test6 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 48, in test6\n    self.assertEqual(patched_source(input_6), original_source(input_6))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test60 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 372, in test60\n    self.assertEqual(patched_source(input_60), original_source(input_60))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test61 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 378, in test61\n    self.assertEqual(patched_source(input_61), original_source(input_61))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test63 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 390, in test63\n    self.assertEqual(patched_source(input_63), original_source(input_63))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test64 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 396, in test64\n    self.assertEqual(patched_source(input_64), original_source(input_64))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test65 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 402, in test65\n    self.assertEqual(patched_source(input_65), original_source(input_65))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test66 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 408, in test66\n    self.assertEqual(patched_source(input_66), original_source(input_66))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test67 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 414, in test67\n    self.assertEqual(patched_source(input_67), original_source(input_67))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test68 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 420, in test68\n    self.assertEqual(patched_source(input_68), original_source(input_68))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test7 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 54, in test7\n    self.assertEqual(patched_source(input_7), original_source(input_7))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test70 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 432, in test70\n    self.assertEqual(patched_source(input_70), original_source(input_70))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test71 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 438, in test71\n    self.assertEqual(patched_source(input_71), original_source(input_71))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test72 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 444, in test72\n    self.assertEqual(patched_source(input_72), original_source(input_72))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test73 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 450, in test73\n    self.assertEqual(patched_source(input_73), original_source(input_73))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test74 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 456, in test74\n    self.assertEqual(patched_source(input_74), original_source(input_74))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test75 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 462, in test75\n    self.assertEqual(patched_source(input_75), original_source(input_75))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test80 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 492, in test80\n    self.assertEqual(patched_source(input_80), original_source(input_80))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test83 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 510, in test83\n    self.assertEqual(patched_source(input_83), original_source(input_83))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test84 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 516, in test84\n    self.assertEqual(patched_source(input_84), original_source(input_84))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test86 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 528, in test86\n    self.assertEqual(patched_source(input_86), original_source(input_86))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test87 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 534, in test87\n    self.assertEqual(patched_source(input_87), original_source(input_87))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test90 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 552, in test90\n    self.assertEqual(patched_source(input_90), original_source(input_90))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test91 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 558, in test91\n    self.assertEqual(patched_source(input_91), original_source(input_91))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test93 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 570, in test93\n    self.assertEqual(patched_source(input_93), original_source(input_93))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test94 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 576, in test94\n    self.assertEqual(patched_source(input_94), original_source(input_94))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test95 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 582, in test95\n    self.assertEqual(patched_source(input_95), original_source(input_95))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n======================================================================\nFAIL: test97 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 594, in test97\n    self.assertEqual(patched_source(input_97), original_source(input_97))\nAssertionError: Lists differ: [1] != [0]\n\nFirst differing element 0:\n1\n0\n\n- [1]\n+ [0]\n\n----------------------------------------------------------------------\nRan 100 tests in 0.016s\n\nFAILED (failures=64)\n')